[2025-05-19T21:00:00.720+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-19T20:00:00+00:00 [queued]>
[2025-05-19T21:00:00.728+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-19T20:00:00+00:00 [queued]>
[2025-05-19T21:00:00.728+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-19T21:00:00.741+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_weather> on 2025-05-19 20:00:00+00:00
[2025-05-19T21:00:00.746+0000] {standard_task_runner.py:60} INFO - Started process 803 to run task
[2025-05-19T21:00:00.749+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_stream_dag', 'fetch_weather', 'scheduled__2025-05-19T20:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/weather_stream_dag.py', '--cfg-path', '/tmp/tmp0z_ec22s']
[2025-05-19T21:00:00.751+0000] {standard_task_runner.py:88} INFO - Job 26: Subtask fetch_weather
[2025-05-19T21:00:00.764+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-19T21:00:00.799+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-19T20:00:00+00:00 [running]> on host f41f956f1644
[2025-05-19T21:00:00.868+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_stream_dag' AIRFLOW_CTX_TASK_ID='fetch_weather' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-19T20:00:00+00:00'
[2025-05-19T21:00:01.018+0000] {logging_mixin.py:188} INFO - [✓] Données météo sauvegardées : data/raw/weather/weather_2025-05-19T21-00-01.json
[2025-05-19T21:00:01.019+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-19T21:00:01.028+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_stream_dag, task_id=fetch_weather, execution_date=20250519T200000, start_date=20250519T210000, end_date=20250519T210001
[2025-05-19T21:00:01.041+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-19T21:00:01.055+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
